sql/pg_duckdb--0.0.1.sql

```diff
@@ -97,14 +97,42 @@ BEGIN
 END;
 $func$;
 
-CREATE OR REPLACE FUNCTION iceberg_scan(path text)
+CREATE OR REPLACE FUNCTION iceberg_scan(path text, allow_moved_paths BOOLEAN DEFAULT FALSE,
+                                                   mode TEXT DEFAULT '',
+                                                   metadata_compression_codec TEXT DEFAULT 'none',
+                                                   skip_schema_inference BOOLEAN DEFAULT FALSE,
+                                                   version TEXT DEFAULT 'version-hint.text',
+                                                   version_name_format TEXT DEFAULT 'v%s%s.metadata.json,%s%s.metadata.json')
```

Comment:
where did you find all these arguments? I only see `allow_moved_paths` in the docs here: https://duckdb.org/docs/extensions/iceberg.html#access-iceberg-metadata

It would be good to add a code comment with a link to the source for these arguments, so we have a reference in case we need to update them in the future.

Reply:
I have checked source code from iceberg extension